run-scripts/run-kiosk-browser-forever-and-log.sh

```diff
@@ -28,6 +28,7 @@ if [ "${VX_MACHINE_TYPE}" = "scan" ]; then
 fi
 
 while true; do
+    sleep 2
```

Comment:
lol

Reply:
in fact I am going to move this to after....but yeah :)